spec/calendar.html

```diff
@@ -564,13 +564,21 @@ <h1>ConsolidateCalendars ( _one_, _two_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-isodaysinmonth" aoid="ISODaysInMonth">
-      <h1>ISODaysInMonth ( _year_, _month_ )</h1>
+    <emu-clause id="sec-temporal-isodaysinmonth" type="abstract operation">
+      <h1>
+        ISODaysInMonth (
+          _year_: an integer,
+          _month_: an integer between 1 and 12 inclusive,
```

Comment:
```suggestion
          _month_: an integer in the inclusive interval from 1 to 12,
```
...is the current terminology in ECMA-262.